website/components/navigation/Navigation.tsx

```diff
@@ -0,0 +1,83 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import TextInput from '@leafygreen-ui/text-input';
+import { spacing } from '@leafygreen-ui/tokens';
+import { SideNav, SideNavGroup, SideNavItem } from '@leafygreen-ui/side-nav';
+import MDBDesignLogo from '../logos/MDBDesignLogo';
+
+const navWidth = css`
+  width: 270px;
+  // spacing[3] already built into side nav
+  padding-left: ${spacing[5] - spacing[3]}px;
+  padding-right: 60px;
+`;
+
+const logoStyles = css`
+  // adds back spacing that was already built into side nav
+  margin-left: ${spacing[3]}px;
+  margin-top: 12px;
+`;
+
+const inputStyles = css`
+  width: 192px;
+  // adds back spacing that was already built into side nav
+  margin-left: ${spacing[3]}px;
+  margin-top: ${spacing[4]}px;
+  margin-bottom: ${spacing[4]}px;
+`;
+
+function Navigation() {
+  return (
+    <nav css={navWidth}>
+      <MDBDesignLogo css={logoStyles} />
+      {/* this is a placeholder until we get our select component */}
+      <TextInput label="" placeholder="Find docs" css={inputStyles} />
+      <SideNav>
```

Comment:
looks like we're nesting `nav` elements. could this have any undesirable effects on tab navigation / screen readers?

Reply:
I haven't seen anything in the spec that suggests that this isn't okay, but if you've seen otherwise I'll definitely update!